keps/sig-instrumentation/1602-structured-logging/README.md

```diff
@@ -621,6 +628,7 @@ _This section must be completed when targeting beta graduation to a release._
   not impact running workloads.
 
 * **What specific metrics should inform a rollback?**
+  N/A
```

Comment:
L626: given you don't use bold anywhere else, let's also not use there?

also nit: move the answer to the new line